profiles/car.lua

```diff
@@ -3,6 +3,7 @@ local find_access_tag = require("lib/access").find_access_tag
 local get_destination = require("lib/destination").get_destination
 local set_classification = require("lib/guidance").set_classification
 local get_turn_lanes = require("lib/guidance").get_turn_lanes
+local canonicalize_spaces = require("lib/stringlist").canonicalize_spaces
```

Comment:
This is a new dependency for `car.lua` and would impact downstream consumers. It needs a changelog entry.